analyses/ssgsea-hallmark/01-run-ssgsea.R

```diff
@@ -7,33 +7,31 @@
 
 #Call libraries
 library("GSVA")
-library("GSEABase");
-
+library("GSEABase")
+library("stringr")
 
 #Read in data 
-expData <- readRDS("../../data/pbta-gene-counts-rsem-expected_count.stranded.rds");
+expData <- readRDS("../../data/pbta-gene-counts-rsem-expected_count.stranded.rds")
 
 #Format RNA-Seq data
-#Get to just one gene symbol per row
-getGene <- function(x)
-{
-    strsplit(x, split="_")[[1]][2]
-}
-expData[,"max"] <- apply(expData[-1], FUN=max, MARGIN=1);
+expData[,"max"] <- apply(expData[-1], FUN=max, MARGIN=1)
```

Comment:
There's a lot of stuff going on here, and it's not obvious to me why these things are happening. Better comments will be important to maintaining this.

Reply:
sure, updated, will push changes shortly.